python/src/spark_rapids_ml/regression.py

```diff
@@ -741,9 +746,16 @@ def _construct_lr() -> CumlT:
 
             return lrs
 
-        def _predict(lr: CumlT, pdf: TransformInputType) -> pd.Series:
-            ret = lr.predict(pdf)
-            return pd.Series(ret)
+        if eval_metric_info:
+
+            def _predict(lr: CumlT, pdf: TransformInputType) -> pd.Series:
```

Comment:
nit: Is the return type CPU pd.Series or a GPU type (e.g. cp.ndarray)?

Reply:
good catch, should be cp.ndarray